frontend/src/views/IntegrationsPage/components/IntegrationsSection/components/IntegrationRow.tsx

```diff
@@ -0,0 +1,190 @@
+import { useMemo } from "react";
```

Comment:
thoughts on sorting this based on the integration by default?

Reply:
I originally had this but my concern is that if an integration failed to sync and it's not on the current page you may miss it; I've pushed a change that will default to ordering by integration unless there are 1 or more failed integrations; then it will sort by status - let me know your thoughts on this approach. Also how do you feel about this styling change for failed integrations?

![CleanShot 2024-12-04 at 17 21 19@2x](https://github.com/user-attachments/assets/ad152560-5a69-49f1-8d7d-f1cba5a899ea)

Reply:
looks good!!!